src/types/location/location.ts

```diff
@@ -45,6 +45,7 @@ export interface LocationList extends LocationBase {
   has_children: boolean;
   parent?: LocationList;
   current_encounter?: Encounter;
+  sort_index?: number;
```

Comment:
is this really optional? 🤔 backend would ALWAYS give a `sort_index` value right? 🤔 

```suggestion
  sort_index: number;
```

Reply:
yes